README.md

```diff
@@ -2,7 +2,7 @@
 This project lets you run a small localhost server that communicates with stellar network.
```

Comment:
- start from h1 (#) instead of h2 (##), update the rest of the headers to match the new hierarchy
- local instead of localhost
- communicate with testnet?

Reply:
👍